### PR TITLE
Added twig example for Actions Menu

### PR DESCRIPTION
**Pull request by eerison**
## Added twig example for Actions Menu

Screenshot 2022-01-13 at 16 16 07

### CODE REVIEW

File: docs/reference/advanced_configuration.rst [hunk 1]
```diff
@@ -386,6 +386,17 @@ overriding the following method::
         return $buttonList;
     }
 
+
+Your custom twig file.::
```
Comment (VincentLanglet) [inline docs/reference/advanced_configuration.rst]:
```suggestion
Your custom twig file::
```

`.:` is weird in your screenshot

File: docs/reference/advanced_configuration.rst [hunk 2]
```diff
@@ -386,6 +386,17 @@ overriding the following method::
...
+
+    {# @App/Button/custom_button.html.twig #}
+    <li>
+     <a href="{{ admin.generateObjectUrl('custom', object) }}">
+         <i class="fa fa-cogs" aria-hidden="true"></i>
+         Custom
+     </a>
+    </li>
```
Comment (VincentLanglet) [inline docs/reference/advanced_configuration.rst]: Can you use 4 spaces and not 2 for the indent ?

File: docs/reference/advanced_configuration.rst [hunk 3]
```diff
@@ -386,6 +386,17 @@ overriding the following method::
         return $buttonList;
     }
 
+
+Your custom twig file.::
+
+    {# @App/Button/custom_button.html.twig #}
```
Comment (VincentLanglet) [inline docs/reference/advanced_configuration.rst:394]:
```suggestion
    {# @App/Button/custom_button.html.twig #}

```

**Review by VincentLanglet — CHANGES_REQUESTED** [review]

**Comment by eerison** [comment]
Screenshot 2022-01-13 at 16 28 25
changed to `code block`

File: docs/reference/advanced_configuration.rst (hunk 3 above)
Comment (eerison) [reply docs/reference/advanced_configuration.rst:394]: I couldn't identify the diference 😄 But I copy and past anyway haha

File: docs/reference/advanced_configuration.rst (hunk 2 above)
Comment (eerison) [reply docs/reference/advanced_configuration.rst]: Done

File: docs/reference/advanced_configuration.rst [hunk 4]
```diff
@@ -370,7 +370,9 @@ Actions Menu
 ------------
 
 You can add custom items to the actions menu for a specific action by
-overriding the following method::
+overriding the following method
+
+.. code-block:: php
```
Comment (VincentLanglet) [inline docs/reference/advanced_configuration.rst]: DOCtor-RST build ask to revert this change

File: docs/reference/advanced_configuration.rst (hunk 3 above)
Comment (VincentLanglet) [reply docs/reference/advanced_configuration.rst:394]: asked by DOCtor-RST build

**Review by VincentLanglet — CHANGES_REQUESTED** [review]

File: docs/reference/advanced_configuration.rst (hunk 1 above)
Comment (eerison) [reply docs/reference/advanced_configuration.rst]: Done

File: docs/reference/advanced_configuration.rst (hunk 4 above)
Comment (eerison) [reply docs/reference/advanced_configuration.rst]: :'( ok then

File: docs/reference/advanced_configuration.rst (hunk 3 above)
Comment (VincentLanglet) [reply docs/reference/advanced_configuration.rst:394]: > I couldn't identify the diference 😄 But I copy and past anyway haha

I added a blank line after the `{# @App/Button/custom_button.html.twig #}`

Comment (eerison) [reply docs/reference/advanced_configuration.rst:394]: I saw, fixed ;) 

how can I run this DOCtor-RST locally ?

**Review by VincentLanglet — APPROVED** [review]

File: docs/reference/advanced_configuration.rst (hunk 3 above)
Comment (VincentLanglet) [reply docs/reference/advanced_configuration.rst:394]: I dunno, the docker image is made by @OskarStark so he should know.

**Review by jordisala1991 — APPROVED** [review]

**Comment by jordisala1991** [comment]
Thanks!
